statistics/selectivity.go

```diff
@@ -479,3 +519,78 @@ func FindPrefixOfIndexByCol(cols []*expression.Column, idxColIDs []int64, cached
 	}
 	return expression.FindPrefixOfIndex(cols, idxColIDs)
 }
+
+// CETraceExpr appends an expression and related information into CE trace
+func CETraceExpr(sc *stmtctx.StatementContext, tableID int64, tp string, expr expression.Expression, rowCount float64) {
+	exprStr, err := ExprToString(expr)
+	if err != nil {
+		return
+	}
```

Comment:
How about printing a debug-level log here so that we can know what types of expressions are not supported yet when testing?

Reply:
Updated.